sycl/include/sycl/platform.hpp

```diff
@@ -144,9 +145,29 @@ class __SYCL_EXPORT platform : public detail::OwnerLessBase<platform> {
   /// Queries this SYCL platform for info.
   ///
   /// The return type depends on information being queried.
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+  template <typename Param>
+  typename detail::is_platform_info_desc<Param>::return_type get_info() const {
+    // For C++11_ABI compatibility, we handle these string Param types
```

Comment:
Nit:
```suggestion
    // For C++11-ABI compatibility, we handle these string Param types
```
This header and `device.hpp` use inconsistent spelling: `C++11_ABI` (first instance) and `C++11-ABI` (second instance).